source/numbers.rst

```diff
@@ -42,6 +42,7 @@ You just type them as literals::
 Questions
 ---------
```

Comment:
> It is so weird that 5/2 returns 2

Python returns the same type in division; its common to use a float e.g `5/2.0` to return `2.5`.